config/1.0.0/earthdata_varinfo_configuration_schema.json

```diff
@@ -29,175 +29,29 @@
         "type": "string"
       }
     },
-    "Excluded_Science_Variables": {
-      "description": "VarInfo classes currently assume that any variable that has a grid mapping attribute, or has a spatial or temporal dimension and is not itself a dimension or bounds variable, should be treated as a science variable. This may not be true in all cases, and so Excluded_Science_Variables provide a method to denote non-science variables that might otherwise be incorrectly identified.",
+    "ExcludedScienceVariables": {
+      "description": "VarInfo classes currently assume that any variable that has a grid mapping attribute, or has a spatial or temporal dimension and is not itself a dimension or bounds variable, should be treated as a science variable. This may not be true in all cases, and so ExcludedScienceVariables provide a method to denote non-science variables that might otherwise be incorrectly identified.",
       "type": "array",
       "items": {
         "$ref": "#/$defs/MissionVariablePatternType"
       }
     },
-    "Required_Fields": {
```

Comment:
Sorry - I meant to add a comment here with the rest. The use of "Fields" here is out of sync with the rest of the schema, which uses "Variables" throughout for the same concept. I wanted to make this more consistent, particularly with `ExcludedScienceVariables`.